<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Java Processes</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>681</width>
     <height>451</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: none;
border: 1px solid rgb(222, 222, 222);
font-size: 14px;
font-weight: 600;
color: rgb(49, 49, 49);
border-color: rgb(49, 49, 49);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="updatePushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>681</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(215, 255, 233);
border: 1px solid rgb(222, 222, 222);
height: 30px;
font-size: 14px;
font-weight: 600;
color: rgb(49, 49, 49);
border-color:rgb(49, 49, 49);</string>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>681</width>
     <height>21</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>470</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: none;
border: 1px solid rgb(222, 222, 222);
height: 30px;
font-size: 14px;
font-weight: 600;
color: rgb(49, 49, 49);
border-color: rgb(49, 49, 49);</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="accelerated">
    <bool>false</bool>
   </property>
   <property name="showGroupSeparator" stdset="0">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>468</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: none;
border: 1px solid rgb(222, 222, 222);
height: 30px;
font-size: 14px;
font-weight: 600;
color: rgb(49, 49, 49);
border-color: rgb(49, 49, 49);</string>
   </property>
   <property name="text">
    <string>Last run time:</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>470</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color:rgb(85, 255, 127);
border: 1px solid rgb(222, 222, 222);
height: 30px;
font-size: 14px;
font-weight: 600;
color: rgb(49, 49, 49);
border-color:rgb(49, 49, 49);</string>
   </property>
   <property name="text">
    <string>Show alternative names</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>434</x>
     <y>470</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(215, 255, 233);
border: 1px solid rgb(222, 222, 222);
height: 30px;
font-size: 14px;
font-weight: 600;
color: rgb(49, 49, 49);
border-color:rgb(49, 49, 49);</string>
   </property>
   <property name="text">
    <string>Show all java applications</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
